hw/ip/spi_host/doc/_index.md

```diff
@@ -198,25 +288,25 @@ Instead the core can be driven by the TL-UL bus clock, and the `sck` period can
 
 #### Chip-select timing control
 
-Typically the `cs_n` line is automatically deasserted after the last edge of `sck`.
-However,  by asserting {{< regref "CONFIGOPTS_0.CSAAT_0" >}} before issuing a particular command, one can instruct the core to hold `cs_n` low indefinitely after the last clock edge.
+Typically the `csb` line is automatically deasserted after the last edge of `sck`.
+However,  by asserting {{< regref "CONFIGOPTS_0.CSAAT_0" >}} before issuing a particular command, one can instruct the core to hold `csb` low indefinitely after the last clock edge.
 This is useful for merging two adjacent commands together, to create very long commands such as continuous read operations.
-The `cs_n` line can then be deasserted by either issuing another command after clearing the {{< regref "CONFIGOPTS_0.CSAAT_0" >}} field, issuing a command to a different device (using a different `COMMAND` register), or simply resetting the core FSM via the {{< regref "CONTROL.RST_FSM">}} register.
+The `csb` line can then be deasserted by either issuing another command after clearing the {{< regref "CONFIGOPTS_0.CSAAT_0" >}} field, issuing a command to a different device (using a different `COMMAND` register), or simply resetting the core FSM via the {{< regref "CONTROL.RST_FSM">}} register.
```

Comment:
The point here is that one could in principle do indefinite reads.  This might not be really that useful for flash (as the software presumably knows how much data it needs), but other devices could really benefit from this.   Also for very long reads the size of a desired read could go beyond the maximum 511 bytes supported.   In which case you'd need to do some stitching.

Reply:
ah - ok that makes sense then